sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/BlobTriggerKind.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Provides blob trigger kinds to detect changes.
+    /// </summary>
+    public enum BlobTriggerKind
+    {
+        /// <summary>
+        /// Changes detection is relied on Storage Analytics logs.
```

Comment:
afaik the analytics scan is combined with container scan (list blobs). should we mention that ? Isn't this strategy falling back to container scan if logs are not there? (I'm not sure about this)

Reply:
You are right, fixed enum value name and the description.